### PR TITLE
DOC: Add registration parameters notebook

### PR DESCRIPTION
**Pull request by thewtex**
@mstaring @dzenanz @romangrothausmann please take a look.

### CODE REVIEW

File: examples/0_HelloRegistrationWorld.ipynb [hunk 1]
```diff
@@ -30,80 +30,41 @@
...
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "fixed = itk.imread('data/CT_2D_head_fixed.mha', itk.F)"
+    "First, let's load our **fixed image** and the image we will align to our fixed image, the **moving image**."
```
Comment (dzenanz) [inline examples/0_HelloRegistrationWorld.ipynb]: Weird sentence

File: examples/2_RegistrationParameters.ipynb [hunk 2]
```diff
@@ -0,0 +1,895 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can run the registration with multiple stages, using the spatial transformation result from the current stage to initialize registration at the next stage.\n",
+    "\n",
+    "Typically, we start with a simple spatial transformation and advance to a more complex spatial transformation.\n",
+    "\n",
+    "The default, conservative, registration parameters progress from a *translation -> affine -> bspline* transformation, with 4 resolutions for each transformation type."
```
Comment (dzenanz) [inline examples/2_RegistrationParameters.ipynb]: translation or rigid?

**Comment by romangrothausmann** [comment]
> @mstaring @dzenanz @romangrothausmann please take a look.

Looks great! Will need some time before I can comment on details (but probably not specific to this PR).

**Comment by romangrothausmann** [comment]
Now better up to date with some details:
How can I get a specific parameter of a specific parameter map, e.g. the `Metric` of the parameter map 2? I.e. how to specify the `ParameterKeyType` of 
http://elastix.isi.uu.nl/doxygen/classelastix_1_1ParameterObject.html#addbd52885b921baa8f737957178b13ec
for translating
https://github.com/romangrothausmann/elastix_scripts/blob/757b3b1add9494fc72aff70d1c85ce3218da0060/recRegStack.py#L241
from SE to ITKElastix?

**Comment by thewtex** [comment]
@romangrothausmann https://github.com/InsightSoftwareConsortium/ITKElastix/pull/38 provides an example.

**Pull request merged**